### PR TITLE
New design

### PR DESCRIPTION
**Pull request by mrushkova**
Changes from https://github.com/vacp2p/vac.dev repo + styles for 404 page
Preview https://vacp2p.netlify.app/

### CODE REVIEW

File: _data/footer-projects.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+- title: "Waku"
+  link: https://status.im/
```
Comment (oskarth) [inline _data/footer-projects.yml:3]: These two links should probably be updated. Assuming things go well with other sites, they should go live at the same time

File: _data/navigation.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+- title: 'Work'
+  link: '/#work'
+  type: local
+
+- title: 'About'
+  link: '/#about'
+  type: local
+
+- title: 'Join VAC'
```
Comment (oskarth) [inline _data/navigation.yml:10]: VAC should be Vac everywhere. Alternatevely, "vac.dev" where it makes sense. We never use VAC with capital letters

File: _data/navigation.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  link: '/#join'
+  type: local
+
+- title: 'Research log'
+  link: '/research-log'
+  type: local
+
+- title: 'Specs'
+  link: 'https://specs.vac.dev'
```
Comment (oskarth) [inline _data/navigation.yml:19]: This should be rfc.vac.dev, it is live there

File: _posts/2019-07-19-p2p-data-sync-for-mobile.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-title:  "P2P Data Sync for Mobile"
-date:   2019-07-19 12:00:00 +0800
+name: 'P2P Data Sync for Mobile'
+title: 'P2P Data Sync for Mobile'
+date: 2019-07-19 12:00:00 +0800
 author: oskarth
 published: true
 permalink: /p2p-data-sync-for-mobile
-categories: research
+category: research
```
Comment (oskarth) [inline _posts/2019-07-19-p2p-data-sync-for-mobile.md:9]: Can we keep this to be categories so it allows for multiple categories?

File: _posts/2019-07-19-p2p-data-sync-for-mobile.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-name:  "P2P Data Sync for Mobile"
-title:  "P2P Data Sync for Mobile"
-date:   2019-07-19 12:00:00 +0800
+name: 'P2P Data Sync for Mobile'
```
Comment (oskarth) [inline _posts/2019-07-19-p2p-data-sync-for-mobile.md:3]: Is there a specific reason we are changing double quotes to single quotes? As opposed to keeping the md files as they are

File: _posts/2019-07-19-p2p-data-sync-for-mobile.md [hunk 6]
```diff
@@ -56,7 +59,7 @@ For requirements or design goals for a solution, here's what we came up with.
 
 ## MVDS - a minimium viable version
 
-The first minimum viable version is in an alpha stage, and it has a [specification](https://rfc.vac.dev/spec/2), [implementation](https://github.com/vacp2p/mvds) and we have deployed it in a [console client](https://github.com/status-im/status-console-client) for end to end functionality. It's heavily inspired by [Bramble Sync Protocol](https://code.briarproject.org/briar/briar-spec/blob/master/protocols/BSP.md).
+The first minimum viable version is in an alpha stage, and it has a [specification](https://github.com/vacp2p/specs/blob/master/mvds.md), [implementation](https://github.com/vacp2p/mvds) and we have deployed it in a [console client](https://github.com/status-im/status-console-client) for end to end functionality. It's heavily inspired by [Bramble Sync Protocol](https://code.briarproject.org/briar/briar-spec/blob/master/protocols/BSP.md).
```
Comment (oskarth) [inline _posts/2019-07-19-p2p-data-sync-for-mobile.md:62]: I think this is an old version of these write-ups, could we use the existing content? Ditto other files

File: _posts/2020-09-28-waku-v2-update.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
 ---
```
Comment (oskarth) [inline _posts/2020-09-28-waku-v2-update.md:1]: I notice assets/img/coscup-waku/walletconnect.png and some other files are deleted - perhaps this is the latest content that hasn't been incorporated?

**Comment by oskarth** [comment]
General things I notice when reading an article such as https://new.vac.dev/presenting-js-waku

1. Twitter card preview seems broken, e.g. see https://cards-dev.twitter.com/validator and compare with new.vac.dev vs vac.dv

2. Article layout so-so. Compare:
 
image
and
image

a) Text is too small on the new one - this is something that I notice throughout the site, can we try to increase this across the board and optimize for scannability/readability?

b) Back button unnecessary

c) Body text isn't centered and it looks weird to have more white space on left than right (probably cause of back button div?)

File: _data/media.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+---
+- title: "[Video] Vitalik dancing badger dance"
```
Comment (oskarth) [inline _data/media.yml:2]: Can we replace these with live ones?

    Media


Some links:

https://www.youtube.com/watch?v=6lLT33tsJjs
https://www.youtube.com/watch?v=lUDy1MoeYnI
https://www.youtube.com/watch?v=s0ATpQ4_XFc
https://www.youtube.com/watch?v=zxN-PqYkZ1M
https://www.youtube.com/watch?v=S782Ppzvkd0
https://www.youtube.com/watch?v=rQOp3qoDF0g

File: _includes/footer.html [hunk 9]
```diff
@@ -1,67 +1,103 @@
...
-                </ul>
-            </div>
-        </div>
+<footer class="footer bg-black flex flex-shrink-0 justify-center">
+  <div
+    class="
+      container
+      max-w-screen-xl
+      flex
+      sl:justify-between
```
Comment (oskarth) [inline _includes/footer.html:7]: Is this code formatting deliberate? Looks a bit odd with a lot of newlines

File: _includes/head.html [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+  <link
+    rel="shortcut icon"
+    href="{{ '/assets/img/favicon.png' | relative_url }}"
+    type="image/png"
+  />
+
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;1,400;1,600&display=swap"
```
Comment (oskarth) [inline _includes/head.html:21]: Do we need to rely on google fonts? We are quite sensitive about analytics etc in general. Is this a big part of the design or can we make do with something simpler?

File: _layouts/default.html [hunk 11]
```diff
@@ -1,79 +1,17 @@
...
-        {% endif %}
-
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <script async defer src="https://cdn.simpleanalytics.io/hello.js"></script>
-        <noscript><img src="https://api.simpleanalytics.io/hello.gif" alt=""></noscript>
-
-        <!-- Fathom - simple website analytics - https://github.com/usefathom/fathom -->
```
Comment (oskarth) [inline _layouts/default.html:20]: Are we keeping the Fathom analytics that is setup?

File: package.json [hunk 12]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "status-vac",
```
Comment (oskarth) [inline package.json:2]: This should be vac.dev

File: package.json [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+  "scripts": {
+    "build:production": "cross-env NODE_ENV=production gulp",
+    "build:dev": "cross-env NODE_ENV=development gulp",
+    "build": "npm run build:production",
+    "dev": "cross-env NODE_ENV=development gulp serve",
+    "start": "npm run dev",
+    "clean": "git clean -fdx",
+    "deploy": "node scripts/deploy.js"
+  },
+  "author": "Ethworks",
```
Comment (oskarth) [inline package.json:29]: Does this mean something specific here? Ideally we keep branding Vac, and then we can mention in README that site was designed by Ethworks

Also missing LICENSE field

File: _data/navigation.yml (hunk 3 above)
Comment (mrushkova) [reply _data/navigation.yml]: Changed

File: _posts/2019-07-19-p2p-data-sync-for-mobile.md (hunk 4 above)
Comment (mrushkova) [reply _posts/2019-07-19-p2p-data-sync-for-mobile.md]: Sure, changed.

File: _posts/2019-07-19-p2p-data-sync-for-mobile.md (hunk 5 above)
Comment (mrushkova) [reply _posts/2019-07-19-p2p-data-sync-for-mobile.md]: Sorry, that was changed by my Prettier settings. Fixed.

File: _includes/footer.html (hunk 9 above)
Comment (mrushkova) [reply _includes/footer.html]: Fixed my Prettier settings

File: tailwind.config.js [hunk 14]
```diff
@@ -0,0 +1,64 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (mrushkova) [reply tailwind.config.js:1]: 🙌🏻

File: _data/media.yml (hunk 8 above)
Comment (mrushkova) [reply _data/media.yml]: Links and data were updated.
Would you like to add media section back?
According to the opened [issue#23](https://github.com/vacp2p/vac.dev/issues) media section and content [were hided](https://github.com/vacp2p/vac.dev/pull/28).
Also new design assumes a preview picture for a video. What pictures for the videos would you like to use?

File: _includes/footer.html [hunk 15]
```diff
@@ -1,67 +1,103 @@
...
+            <ul>
+              {% for nav in site.data.footer-projects %} {% include
+              nav-item-footer.html %} {% endfor %}
+            </ul>
+          </div>
         </div>
+        {%- endcomment -%}
+      </nav>
 
+      {%- comment -%}
```
[the remark this answers is not in the document]
Comment (mrushkova) [reply _includes/footer.html:36]: My PRs in [vac.dev](https://github.com/vacp2p/vac.dev/pulls) repo are not merged and thats why are not applied.
You should add script for a signup form after adding some email list.

File: _data/media.yml (hunk 8 above)
Comment (oskarth) [reply _data/media.yml]: Oh right, miscommunication between me and Jonny I think. Yes please, we can use whatever preview is default in youtube or so I think

File: package.json (hunk 12 above)
Comment (oskarth) [reply package.json]: Talked about in Discord

File: package.json (hunk 13 above)
Comment (oskarth) [reply package.json]: sure, thanks

File: _data/media.yml (hunk 8 above)
Comment (mrushkova) [reply _data/media.yml]: Should the video open as a modal on the page, or should it just be a YouTube link and opened in a new tab?

File: package.json (hunk 12 above)
Comment (mrushkova) [reply package.json]: [New design](https://vacp2p.netlify.app/presenting-js-waku) for articles is applied

**Comment by oskarth** [comment]
@mrushkova would you mind rebasing on master this to get latest article included? Then we can merge and tackle any other issues in follow up issues/PRs

**Pull request merged**